### PR TITLE
feat: mirroring documentation

### PR DESCRIPTION
**Pull request by lleyton**

### CODE REVIEW

File: src/content/docs/en/contributing/mirroring.mdx [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
+title: Mirroring
+description: "A guide of how you can mirror Ultramarine Linux, making downloads faster for everyone"
+---
+
+Mirroring allows you to contribute to the Ultramarine Linux ecosystem by hosting packages closer to users, making downloads faster, more reliable, and reducing load on Fyra Labs' core infrastructure.
```
Comment (nothingneko) [inline src/content/docs/en/contributing/mirroring.mdx:6]:
```suggestion
Mirroring allows you to contribute  by hosting packages closer to users, making downloads faster, more reliable, and reducing load on Fyra Labs' infrastructure.
```

File: src/content/docs/en/contributing/mirroring.mdx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+When users update or install packages, they'll hit our mirror director service, [Tetsudou](https://github.com/terrapkg/tetsudou), which will calculate an optimal set of mirrors to download updates from, possibly including yours.
```
Comment (nothingneko) [inline src/content/docs/en/contributing/mirroring.mdx:8]:
```suggestion
When users update or install packages, they'll hit our mirror director service, [Tetsudou](https://github.com/terrapkg/tetsudou), which will calculate an optimal set of mirrors to download updates from.
```
good idea but feels kinda forced here

File: src/content/docs/en/contributing/mirroring.mdx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+You can also mirror the repositories using the `rsync-ssl` utility.
+
+```sh
+rsync-ssl -avPzr --delete rsync://repos.fyralabs.com/repo/ /path/to/your/mirror
+```
+
+## Running on a schedule
+
+Regardless of whether you're using the mirror script or directly calling rsync, you'll need to run them on a schedule to keep your mirror up-to-date. We recommend schedudling for every 5 minutes.
```
Comment (nothingneko) [inline src/content/docs/en/contributing/mirroring.mdx:32]:
```suggestion
Regardless of whether you're using the mirror script or directly calling rsync, you'll need to rerun to keep your mirror up-to-date. We recommend syncing every 5 minutes.
```

File: src/content/docs/en/contributing/mirroring.mdx [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+This is required if you want to submit your mirror to the public list. We may remove mirrors that are not kept up-to-date.
```
Comment (nothingneko) [inline src/content/docs/en/contributing/mirroring.mdx:34]:
```suggestion
This is required if you want to submit your mirror to the public list. We will remove mirrors that are not kept up-to-date.
```
certainty is better here

File: src/content/docs/en/contributing/mirroring.mdx [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+## Serving your mirror
+
+All mirrors must be served over HTTPS. You can use a web server of your choosing to serve your mirror, we recommend using [Static Web Server](https://static-web-server.net/) or [Caddy](https://caddyserver.com/).
+
+## Submitting your mirror
+
+Once you have your mirror set up, you can submit it to the public mirror list by opening a pull request on the [Tetsudou repository](https://github.com/terrapkg/tetsudou).
+
+The public mirror configuration is stored in the `repos/` directory. Each JSON file represents a repository that can be mirrored. You can add your mirror in the appropriate JSON files.
+If you're mirroring all repositories, the default, you ned to add your mirror to all JSON files.
```
Comment (madonuko) [inline src/content/docs/en/contributing/mirroring.mdx:47]:
```suggestion
If you're mirroring all repositories (the default), you ned to add your mirror to all JSON files.
```

**Pull request merged**